app/controllers/application_controller.rb

```diff
@@ -104,4 +104,11 @@ def authenticate_and_check_access
       render "errors/access_denied", status: :forbidden, formats: :html
     end
   end
+
+  def non_maintenance_bypass_ip_address?
+    return true if Settings.maintenance_mode.bypass_ips.blank?
+
+    bypass_ip_list = Settings.maintenance_mode.bypass_ips.split(",").map { |ip| IPAddr.new ip.strip }
```

Comment:
You could maybe move `Settings.maintenance_mode.bypass_ips` into a variable
```suggestion
    bypass_ips = Settings.maintenance_mode.bypass_ips
    return true if bypass_ips.blank?

    bypass_ip_list = bypass_ips.split(",").map { |ip| IPAddr.new ip.strip }
```

Reply:
I've made this change